cli/cli_click.go

```diff
@@ -2,28 +2,81 @@ package cli
 
 import (
 	"bufio"
+	"context"
+	"errors"
 	"fmt"
+	"net/url"
 	"os"
+	"time"
 
+	"github.com/NordSecurity/nordvpn-linux/daemon/pb"
+	"github.com/NordSecurity/nordvpn-linux/internal"
+	"github.com/fatih/color"
 	"github.com/urfave/cli/v2"
 )
 
-// Click is hidden cmd used to open some info when clicking on desktop icon
-func (c *cmd) Click(ctx *cli.Context) error {
-	if ctx.NArg() >= 1 {
-		// if arg is given
-		// run the same as: login --callback %arg
-		return c.oauth2(ctx)
+func waitForInput(timeout bool) {
+	inputChan := make(chan interface{})
+	go func() {
+		bufio.NewReader(os.Stdin).ReadBytes('\n')
+		close(inputChan)
+	}()
+
+	if !timeout {
+		<-inputChan
+		return
 	}
 
-	if err := cli.ShowAppHelp(ctx); err != nil {
-		return err
+	select {
+	case <-inputChan:
+	case <-time.After(10 * time.Second):
```

Comment:
This timeout should be discussed: 10 seconds might be not enough for user to take action, and after 10 seconds - terminal window is closed? or at least no user input possibility.